pubspec.yaml

```diff
@@ -7,7 +7,8 @@ environment:
   sdk: ^3.8.0
 
 dependencies:
-  kiss_repository: ^0.10.0
+  kiss_repository:
+    path: ../kiss_repository
```

Comment:
why?

Reply:
will update to 0.11.0, once we publish it. otherwise dependency resolution will fail, due to circular dependencies